### DISCUSSION TITLE
Drop down to order SQL snippets by A-Z

### DISCUSSION THREAD

**Discussion by garylake** [supabase/supabase#13824] [Feature Requests]
Recently the SQL snippets side column was ordered A-Z. A recent update seems to have made this ordered by created_at or maybe updated_at? 

I'm sure there was logic behind this but I manage quite a lot of database functions via SQL snippets and the move away from A-Z ordering is giving me productivity hell 😅

I appreciate others might like this date ordering so could we get a simple drop down for 'order by' for the snippets list. E.g. A-Z asc/desc, created asc/desc, updated asc/desc, etc.

**Comment by garylake** [comment]
@saltcod

**Reply by saltcod** [reply]
Thanks @garylake !

**Reply by gabrielsestrem** [reply]
Please order by A-Z asc/desc

**Comment by gabrielsestrem** [comment]
Hi! I really need this too. I have 98 SQL snippets and we use this for many reasons to collect data and make decisions in our company. Please order from A-Z it's really important to us.

**Comment by jean-c-martins** [comment]
Ordering the queries by A-Z was so much better, not sure why they changed it but  we must have the option to order it by A-Z if we want to.

**Comment by MarcosNespolo** [comment]
Such changes can have serious implications for many applications. Supabase should be more careful about how minor modifications can influence their users.

**Comment by saltcod** [comment]
Hey all — apologies on this one. It was an unintended consequence of a recent refactor. We've restored the alpha sort order — will be deployed this week. 

Thanks for the feedback.

**Reply by gabrielsestrem** [reply]
Thanks @saltcod  it's working perfectly now! cheers

**Comment by garylake** [comment]
🎉🎉🎉

Thanks @saltcod!
